modules/social_features/social_event/social_event.module

```diff
@@ -498,6 +498,14 @@ function social_event_block_access(Block $block, $operation, AccountInterface $a
  */
 function social_event_form_alter(&$form, FormStateInterface $form_state, $form_id) {
   social_event_show_pending_requests_message();
+
+  if (in_array($form_id, [
```

Comment:
This should go in the switch below. I can help with that.

Reply:
I merged the `if` and `switch` statements in 46090fa9e01362edc51e1335e9b998cc50c2671a